validation-test/stdlib/String.swift

```diff
@@ -963,7 +963,7 @@ StringTests.test("growth") {
     s += "x"
     s2 = s
   }
-  expectLE(s.nativeCapacity, 34)
+  expectLE(s.nativeCapacity, 40)
```

Comment:
I'll remove this change from this pull request. My intention for this pull request is to get the tests to run, *not* to get them to all pass.